src/components/Identity/index.js

```diff
@@ -106,6 +106,7 @@ const createIdentity = ({ config, logger, cookie }) => {
 };
 
 createIdentity.namespace = "Identity";
+createIdentity.cookieNamespace = "I";
```

Comment:
It might be good to rename `cookieNamespace` to `abbreviation` so it could potentially be used in other contexts. 🤷‍♂ We can rename it later if needed.